io.catenax.part_as_specified/2.0.0/PartAsSpecified.ttl

```diff
@@ -0,0 +1,139 @@
+#######################################################################
+# Copyright (c) 2022 T-Systems International GmbH
+# Copyright (c) 2022 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+# Copyright (c) 2022 Volkswagen AG
+# Copyright (c) 2022 Robert Bosch GmbH
+# Copyright (c) 2022 ZF Friedrichshafen AG
+# Copyright (c) 2022 Contributors to the Eclipse Foundation
+#
```

Comment:
check who needs to be added here after the changes

Reply:
we will check this asap and update this line after our check

Reply:
we checked this and updated as needed